README.md

```diff
@@ -0,0 +1,29 @@
+Public Transport Enabler
+========================
+
+This is a Java library allowing you to get data from public transport providers.
+Look into [AbstractNetworkProvider.java](https://github.com/schildbach/public-transport-enabler/blob/master/enabler/src/de/schildbach/pte/AbstractNetworkProvider.java) for an overview of the API.
+
+Using providers that require secrets
+------------------------------------
+
+For some providers a secret like an API key is required to use their API.
+Copy the `secrets.properties.template` file to `secrets.properties` like so:
+
+    $ cp enabler/test/de/schildbach/pte/live/secrets.properties.template enabler/test/de/schildbach/pte/live/secrets.properties
+
+You need to request the secrets directly from the provider. For Navitia based providers, you can [request a secret here](http://www.navitia.io/register).
+
+How to run live tests?
+----------------------
+
+Make sure the test you want to run does not require a secret and if it does, see above for how to get one.
```

Comment:
I think the straightforward way to run single tests is to run them via your IDE. Both IntelliJ and Eclipse have excellent support for JUnit tests.